graphspace_python/api/endpoint/graphs.py

```diff
@@ -109,12 +122,14 @@ def delete_graph(self, name):
 
 		:return: Success/Error Message from GraphSpace
 		"""
-		graph = self.get_graph(name)
-		if graph is None or 'id' not in graph:
+		response = self.get_graph(name)
+		if response is None or response.graph.id is None:
 			raise Exception('Graph with name `%s` doesnt exist for user `%s`!' % (name, self.client.username))
 		else:
-			graph_by_id_path = GRAPHS_PATH + str(graph['id'])
-			return self.client._make_request("DELETE", graph_by_id_path).json()
+			graph_by_id_path = GRAPHS_PATH + str(response.graph.id)
```

Comment:
There is no need to create a variable for this. We can directly append id to paths while making requests.